app/assets/javascripts/input_validation.js

```diff
@@ -0,0 +1,5 @@
+$(function(){
+  $(document).on("change", ".field_with_errors", function() {
```

Comment:
Interesting, does this get triggered when the DOM of `.field_with_errors` changes?